docs/extras/modules/model_io/output_parsers/xml.ipynb

```diff
@@ -0,0 +1,178 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "181b5b6d",
+   "metadata": {},
+   "source": [
+    "# XML parser\n",
+    "This output parser allows users to obtain results from LLM in the popular XML format. \n",
+    "\n",
+    "Keep in mind that large language models are leaky abstractions! You'll have to use an LLM with sufficient capacity to generate well-formed XML."
```

Comment:
I sent you API token for Claude 2, it would be good to mention & have example based on it. 
Anthropic seems to suggest to use XML in the docs as you can see https://docs.anthropic.com/claude/docs/optimizing-your-prompt that XML tags are preferred method for the model - this is why it is interesting feature to have.